### ISSUE TITLE
site is down

### ISSUE THREAD

**Issue by ansuz** [zielmicha/fc00.org#18]

**Comment by stefancrain** [comment]
I also see the site as "down", here's some debugging info:
- I'm not connected to cjdns
- In chrome I see an nginx error
- The site resolves to 87.98.234.72

![image](https://cloud.githubusercontent.com/assets/330720/20653913/3aef306c-b4e2-11e6-9082-138201fb09de.png)

```
$ dig fc00.org

; <<>> DiG 9.8.3-P1 <<>> fc00.org
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 28087
;; flags: qr rd ra; QUERY: 1, ANSWER: 1, AUTHORITY: 0, ADDITIONAL: 0

;; QUESTION SECTION:
;fc00.org.			IN	A

;; ANSWER SECTION:
fc00.org.		3589	IN	A	87.98.234.72

;; Query time: 32 msec
;; SERVER: 8.8.8.8#53(8.8.8.8)
;; WHEN: Sun Nov 27 20:41:35 2016
;; MSG SIZE  rcvd: 42
```

**Comment by zielmicha** [comment]
Fixed.
